hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/bucket/S3BucketCreateRequest.java

```diff
@@ -214,7 +214,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
               transactionLogIndex));
 
       OMBucketCreateResponse omBucketCreateResponse =
-          new OMBucketCreateResponse(omBucketInfo, omResponse.build());
```

Comment:
One question why there is no such logic of replay for S3Bucket Create Request.

Reply:
Missed it. Thanks for catching. I will update it.